### DISCUSSION TITLE
Can we change Assistant Language?

### DISCUSSION THREAD

**Discussion by VasigaranAndAngel** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#459] [Q&A]
Can we change assistant language? (Not UI language)
If can, How?
If can not, then is this feature can be added to this project?

**Comment by sumeshir26** [comment]
@VasigaranAndAngel Yes, you can. Just ask your query in that language and the Assistant will automatically respond in that language.

**Reply by VasigaranAndAngel** [reply]
![image](https://user-images.githubusercontent.com/72515046/126957584-ad6036ce-3085-4ff7-83f7-34471c360701.png)
I'm asking for change the interacting language of assistant... 😶😶

**Reply by im-coder-lg** [reply]
> I'm asking for change the interacting language of assistant...

Simple. If you say `Talk in {language}`(replace {language} with your choice) then Google Assistant will respond in that language. The same way it does for mobile.
**Note: It has to be supported by the GAUDC Client.**
